### DISCUSSION TITLE
Prisma instrumentation 5.0.0 change/release notes

### DISCUSSION THREAD

**Discussion by paymog** [prisma/prisma#20401] [Q&A] [answered]
I'm using `@prisma/instrumentation` in my application. I recently upgraded prisma and prisma client to 5.0.0. Should I also update `@prisma/instrumentation` to version 5.0.0? I can't seem to find reliable release/change notes for this npm package to determine if it should be upgraded as well (and what breaking changes might be introduced since it doesn't follow semver).

**Comment by Jolg42** [comment, marked as answer]
@paymog You should keep it the same version for simplicity

Technically speaking, in that release we didn’t do any changes to it, except keep the dependencies up-to-date
See https://github.com/prisma/prisma/commits/5.0.0/packages/instrumentation
